### PR TITLE
as far as i've gotten

### PR DESCRIPTION
**Pull request by mohamedrchalal**
havent finished resume.html

### CODE REVIEW

File: Resume.HTML [hunk 1]
```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<head>
+  <title>Wendy Bite | Resume</title>
+  <meta charset="utf8">
+</head>
+<style>
```
Comment (nolds9) [inline Resume.HTML]: How could we move our styles out of the `style` tag so to practice separation of concerns?Also, I think many of the styles here are repetive / are being overriden by the external stylesheet...

File: Style.css [hunk 2]
```diff
@@ -0,0 +1,66 @@
+body{
```
Comment (nolds9) [inline Style.css]: In the future, I recommend time-boxing looking at the solution so that you can get some practice

File: index.html [hunk 3]
```diff
@@ -1,7 +1,9 @@
 <!DOCTYPE html>
+<html>
 <head>
   <title>Wendy Bite | About Me</title>
   <meta charset="utf8">
+  <link type="text/css" rel="stylesheet" href="style.css" />
 </head>
 <body>
```
Comment (nolds9) [inline index.html]: would love to see you come back to this and complete the `About Me` section

**Comment by nolds9** [comment]
:+1:  Thanks for submitting what you have, in the future please work to timebox looking at the solution, rather use it as a jumping of point. Also please include relevant `completion` and `comfort level` scores on a scale of 5, as well as feel free to leave any questions or areas of code where you want specific feedback.

**Comment by mohamedrchalal** [comment]
Will do, thank you. This exercise and CSS in genereal were extremely
difficult for me. I have been watching CSS tutorials for the past hour or
so in hopes to have a firmer grasp on it. Thank you for the feedback!

**Pull request closed without merging**